packaging/Dockerfile.centos_7_git_rpm

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is based on the recommendations provided in the
```

Comment:
The name `git_rpm` is a bit strange. If I understand correctly, this image contains master. So you could use `master` or `master_copr`.

Reply:
IMHO Copr is an implementation detail that is not relevant for the image name. `git` is the name that we use in the Copr repo, i.e. `nmstate-git-el7` - so should we rename that copr to `nmstate-git-master-el7`?